p2p/discovery.py

```diff
@@ -820,6 +946,63 @@ def __init__(self, proto: DiscoveryProtocol, peer_pool: PeerPool,
         await self.proto.stop()
 
 
+class NodeTicketInfo:
+    last_issued: int = 0
+    last_used: int = 0
```

Comment:
These could potentially use some comments explaining the purpose of both of these fields.  Reading the code I haven't figured out exactly what they are yet.

Reply:
```
class NodeTicketInfo:
    # The serial number of the last ticket we issued for a given remote node. New tickets are
    # issued when a node sends a PING msg containing one or more topics.
    last_issued: int = 0
    # The serial number of the last ticket used by a given remote node. Tickets are marked as used
    # when a node sends a REGISTER_TICKET msg using a ticket previously issued.
    last_used: int = 0

```